staging/src/k8s.io/apiextensions-apiserver/pkg/cmd/server/options/options.go

```diff
@@ -43,9 +45,10 @@ const defaultEtcdPathPrefix = "/registry/apiextensions.kubernetes.io"
 
 // CustomResourceDefinitionsServerOptions describes the runtime options of an apiextensions-apiserver.
 type CustomResourceDefinitionsServerOptions struct {
-	ServerRunOptions   *genericoptions.ServerRunOptions
-	RecommendedOptions *genericoptions.RecommendedOptions
-	APIEnablement      *genericoptions.APIEnablementOptions
+	ServerRunOptions     *genericoptions.ServerRunOptions
+	RecommendedOptions   *genericoptions.RecommendedOptions
+	APIEnablement        *genericoptions.APIEnablementOptions
+	TransformerOverrides map[schema.GroupResource]value.Transformer
```

Comment:
`CustomResourceDefinitionsServerOptions` is not a struct that this PR should be changing.